CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## UNRELEASED
+
+- Added support for PSR-17 factories.
+- Dropped support for PHP < 7.2
```

Comment:
this would now have been 7.1

Reply:
Oh, sorry.

Reply:
https://github.com/php-http/multipart-stream-builder/pull/40